lit_gpt/data/alpaca.py

```diff
@@ -109,6 +109,16 @@ def val_dataloader(self) -> DataLoader:
             collate_fn=get_sft_collate_fn(max_seq_length=self.max_seq_length, ignore_index=self.ignore_index)
         )
 
+    def __repr__(self) -> str:
```

Comment:
maybe not for this PR, but we should support datamodule dataclasses: https://github.com/Lightning-AI/pytorch-lightning/pull/9039